### DISCUSSION TITLE
Trash support

### DISCUSSION THREAD

**Discussion by 0323pin** [woelper/oculante#230] [General]
Hi,

Just thought about keeping this alive.

Still have some digging to do and, I didn't forget this but, it will take a few days. Perhaps, during the weekend, I think I'm close enough, https://github.com/Byron/trash-rs/pull/84

I want to try a few things before doing a PR.

Thanks for excluding it for now.

**Comment by Stoppedpuma** [comment]
This was implemented at some point so I'll close this. Thanks!
